generic_config_updater/services_validator.py

```diff
@@ -17,13 +17,42 @@ def set_verbose(verbose=False):
 
 def _service_restart(svc_name):
     rc = os.system(f"systemctl restart {svc_name}")
-    logger.log(logger.LOG_PRIORITY_NOTICE,
-            f"Restarted {svc_name}", print_to_console)
+    if rc != 0:
+        # This failure is likely due to too many restarts
+        #
+        rc = os.system(f"systemctl reset-failed {svc_name}")
+        logger.log(logger.LOG_PRIORITY_ERROR, 
+                f"Service has been reset. rc={rc}; Try restart again...",
+                print_to_console)
+
+        rc = os.system(f"systemctl restart {svc_name}")
+        if rc != 0:
```

Comment:
>rc [](http://example.com/codeflow?start=11&length=2)

Is the rc a stable code when 2nd restart fail?

Reply:
I don't get your comments. But rc is the absolute way to check success / failure.